internal/screen/screen.go

```diff
@@ -141,8 +141,16 @@ func TempStart(screenWasNil bool) {
 func Init() error {
 	drawChan = make(chan bool, 8)
 
+	// Is true color support available?
+	truecolorAvailable := (os.Getenv("COLORTERM") == "truecolor") ||
+		(os.Getenv("COLORTERM") == "24bit") ||
+		(os.Getenv("COLORTERM") == "24-bit")
```

Comment:
> So no need to do it again in micro itself.

Yes, due to the fact that it's already done within tcell there is no need to do it twice. I fully confirm. Most probably I should have checked tcell earlier too by following the flag...
My fault.

> Although your PR adds possibility to use an option instead of env vars for that, [...]

Hm, ok...then it's most probably no longer needed at all. Then I would change my mind to better change the default of the `MICRO_TRUECOLOR` (use it to explicitly disable TC) handling, since I don't like the env var approach to enable that feature.

Reply:
Upon more thought, maybe actually it would be nice to have the `truecolor` option (replacing the `MICRO_TRUECOLOR` variable) as a single interface point for controlling the truecolor behavior, which would merely do the following: `off` value sets `TCELL_TRUECOLOR=disable` to disable truecolor, `on` sets `COLORTERM=truecolor` to force truecolor, `auto` does nothing.

Reply:
Done, but didn't remove `MICRO_TRUECOLOR` yet.

Reply:
If we introduce the new method while keep the old method, IMO things become confusing: need to specify what overrides what, and so on.

Let's try not to introduce more mess.

Let's first figure out what exactly do we want. Starting from the question do we want autodetection enabled by default or not, and why.

Reply:
I will vote for the activation of the auto detection by default, because it reduces the necessary configuration and the trouble by switching between different types of color/highlighting schemes.
Furthermore the configuration interface via `MICRO_TRUECOLOR` should be declared deprecated to directly removed, to return to the single point of configuration.